packages/platform-test-suite/.env.example

```diff
@@ -16,3 +16,4 @@ DASHPAY_OWNER_PRIVATE_KEY=
 # MASTERNODE_REWARD_SHARES_OWNER_PRO_REG_TX_HASH=
 # MASTERNODE_REWARD_SHARES_OWNER_PRIVATE_KEY=
 # MASTERNODE_REWARD_SHARES_MN_OWNER_PRIVATE_KEY=
+FAUCET_WALLET_USE_STORAGE=
```

Comment:
This makes this variable mandatory. Let's have it commented for information purpose.

```suggestion
# Enable faucet wallet storage
# FAUCET_WALLET_USE_STORAGE=true
```

Reply:
Done